app/__init__.py

```diff
@@ -14,13 +14,16 @@ def create_app(test_config=None):
     app = Flask(__name__)
     app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 
-    if test_config is None:
-        app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
-            "SQLALCHEMY_DATABASE_URI")
-    else:
-        app.config["TESTING"] = True
-        app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
-            "SQLALCHEMY_TEST_DATABASE_URI")
+    app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get(
+        "RENDER_DATABASE_URI")
+
+    # if test_config is None:
```

Comment:
I'd recommend using this logic to switch between testing and non-testing environments. This is why your tests are erroring out in Learn.